SzczepionkaAPI/src/main/java/com/szczepionka/controller/EmailController.java

```diff
@@ -0,0 +1,32 @@
+package com.szczepionka.controller;
+
+import javax.mail.MessagingException;
+import javax.mail.internet.AddressException;
+
+import com.szczepionka.model.PatientDTO;
+import com.szczepionka.service.EmailService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.*;
+
+import java.io.IOException;
+
+@RestController
+public class EmailController {
+
+    public final EmailService emailService;
+
+    public EmailController(EmailService emailService) {
+        this.emailService = emailService;
+    }
+
+
```

Comment:
Try to press Ctrl+Alt+L to autoformat your code. I think that it's needed in every class.

Reply:
fixed